docs/reference/setup/sysconfig/virtual-memory.asciidoc

```diff
@@ -1,8 +1,8 @@
 [[vm-max-map-count]]
 === Virtual memory
 
-Elasticsearch uses a <<default_fs,`hybrid mmapfs / niofs`>> directory by
-default to store its indices.  The default operating system limits on mmap
+Elasticsearch uses a <<default_fs,`mmapfs`>> directory by
+default for 64bit systems to store its indices.  The default operating system limits on mmap
```

Comment:
This is a good call, I opened https://github.com/elastic/elasticsearch/pull/25971. In the meantime I see your PR targets 5.5 where we still support 32-bit systems on a best-effort so I'll merge it as-is and just remove the 64-bit reference when merging to the 6.x/7.x branches.